packages/editor/src/components/widgets/table/cell/EditableCell.tsx

```diff
@@ -1,7 +1,11 @@
 import './EditableCell.css';
+import '../../code-editor/ScriptInput.css';
 import { CellContext, RowData } from '@tanstack/react-table';
 import { useEffect, useState } from 'react';
 import { Input } from '../../input';
+import { Browser, useBrowser } from '../../../../components/browser';
+import { usePath } from '../../../../context';
+import { useCodeEditorOnFocus } from '../../code-editor/useCodeEditor';
```

Comment:
looks like this hook no longer has the correct name nor location.
I think it should live in the browser folder and should be called useInputOnFocus or useOnFocus or useOnInputFocus or something the like